tests/generics/twrong_generic_object.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  errormsg: "cannot instantiate: 'GenericNodeObj'"
+  errormsg: "cannot instantiate generic : 'GenericNodeObj[T]'; Maybe generic arguments are missing?"
```

Comment:
1st `generic` seems redundant now that u've added both [T] and `Maybe generic arguments are missing` ; maybe:
`errormsg: "cannot instantiate generic : 'GenericNodeObj[T]'; Maybe generic arguments are missing?"`
=>
`errormsg: "cannot instantiate: 'GenericNodeObj[T]'; Maybe generic arguments are missing?"`